### PR TITLE
Update Readme.md for L10N Updates

### PR DESCRIPTION
**Pull request by ecotopian**
@vauxhall  We both worked today on L10N with transifex - could be I overwrote yours 🙈 I adjust the workflow to also pull the source file before pushing it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,10 +184,12 @@ Test and commit the new translations. There exists schemes for testing other lan
...
 #### Via CLI
 
-1. Add the new strings to the English source file `Client/Ecosia/L10N/en.lproj/Ecosia.strings`
-2. Push it to Transifex
+1. Pull the source file
+2. Add the new strings to the English source file `Client/Ecosia/L10N/en.lproj/Ecosia.strings`
+3. Push it to Transifex
 
 ```bash
+tx pull -s
```
Comment (vauxhall) [inline README.md:192]: isn't is `tx pull -af` ?

**Review by vauxhall — APPROVED** [review]
I think this change is good.

However, for the way we handle translations is not possible to have this kind of merge conflicts. We use English as the base language and we never pull English. The other languages are generated based on the base language. Does it make sense?

File: README.md (hunk 1 above)
Comment (ecotopian) [reply README.md:192]: yes true, but you miss the source file (en) when you don't append `-s` . So to be super accurate it should be `tx pull -afs` when you want to force all the changes from remote

**Comment by ecotopian** [comment]
> I think this change is good.
> 
> However, for the way we handle translations is not possible to have this kind of merge conflicts. We use English as the base language and we never pull English. The other languages are generated based on the base language. Does it make sense?

if you work alone it makes sense but causes problems if translation are added in two different branches. e.g. if we never pull English, pushing it will delete keys that you don't have and were pushed from another branch. So I think it's mandatory that we should pull English before adding new keys to it.

**Comment by vauxhall** [comment]
> > I think this change is good.
> > However, for the way we handle translations is not possible to have this kind of merge conflicts. We use English as the base language and we never pull English. The other languages are generated based on the base language. Does it make sense?
> 
> if you work alone it makes sense but causes problems if translations are added in two different branches. e.g. if we never pull English, pushing it will delete keys that you don't have and might have been pushed from another branch. So I think it's mandatory that we should pull English before adding new keys to it.

They would not be deleted from the repository as git takes care of the merge, but there might be conflicts for the translators, assuming they translate faster than we merge branches, so I think is a good suggestion 👍

File: README.md (hunk 1 above)
Comment (vauxhall) [reply README.md:192]: @ecotopian  Then I think it would make sense to update the documentation to `tx pull -afs`

**Pull request merged**